### PR TITLE
Address okwme comments

### PR DESCRIPTION
**Pull request by ethanfrey**
This addresses all outstanding comments in the various PRs submitted by @okwme 

Thanks for the deep review of the tutorial.

### CODE REVIEW

File: docs/intro/rust-basics.md [hunk 1]
```diff
@@ -86,14 +86,17 @@ Now that you can compile and test the code, it is time to edit it. But before th
...
 
 [RUST for Intellij](https://intellij-rust.github.io/)
 
 There are many more editors out there and some have varying degrees of rust support, at least syntax highlighting, but I would recommend trying one of the two above, especially if you are new to rust. Once you are confident in the language, you can always use another editor and customize it to your liking.
 
+### Setting Root Directory
+
+Both of the above extensions look for a Cargo.toml file in the root directory of your workspace, and ony parse rust code referenced by this Cargo.toml file (listed as a workspace, or imported by `src/lib.rs`). The [`cosmwasm-examples`](https://github.com/confio/cosmwasm-examples) repo does not have a `Cargo.toml` file, but rather one in each example sub-directory. To ensure proper IDE support when working on this example, you should open only the `escrow` directory. And in general, have one window open for one rust projects, rooted in the same directory as it's `Cargo.toml` file.
```
Comment (okwme) [inline docs/intro/rust-basics.md]: "and ony parse rust" >> "and only parse rust"

File: docs/intro/first-demo.md [hunk 2]
```diff
@@ -41,22 +41,25 @@ wasmcli query account $(wasmcli keys show bob -a)
...
 # Set the THIEF variable in source code to this value
 wasmcli keys show thief -a
 
 # and recompile wasm
 cd <path/to/rust/code>
 docker run --rm -u $(id -u):$(id -g) -v $(pwd):/code confio/cosmwasm-opt:0.4.1
 ls -lh contract.wasm
-cp contract.wasm $HOME [__Why is this necessary?__]
+
+# (optional) let's go back to where you were running ($HOME ?), not run inside the rust code dir
```
Comment (okwme) [inline docs/intro/first-demo.md]: I think I was running everything from within the rust code dir. Is that a problem?

**Review by okwme — APPROVED** [review]
LGTM!
Just the one comment about running code from the rust dir.
Can't believe there were so many spelling mistakes still there! Did you run a spellcheck on it or sth?

File: docs/intro/first-demo.md (hunk 2 above)
Comment (ethanfrey) [reply docs/intro/first-demo.md]: No problem, I think I said earlier to run everything in $HOME
Mainly not to add download.wasm to the rust dir.

But maybe you are right and it is simpler to do it all next to the source code. No cd needed.

I will update this

**Comment by ethanfrey** [comment]
I finally installed a spellcheck plugin to VSCode. Something I should have done years ago

**Pull request merged**